pkg/config/config_test.go

```diff
@@ -44,16 +44,16 @@ func TestConfigBlacklist(t *testing.T) {
 }
 
 func TestLoadExampleConfig(t *testing.T) {
-	config, err := LoadConfig("test-fixtures/example.yaml")
+	config, err := Load("test-fixtures/example.yaml")
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	expect := NukeConfig{
+	expect := Nuke{
```

Comment:
without the "*Config" it feels like it's missing context, because the package it is in, is not something i'd look at first when guessing what a struct does. just my 2 cents...

Reply:
This is convention: https://github.com/golang/go/wiki/CodeReviewComments#package-names